### DISCUSSION TITLE
Add health check endpoint to the backend

### DISCUSSION THREAD

**Discussion by jirutka** [lukevella/rallly#1133] [Ideas]
Can you please add a simple health check GET/HEAD endpoint to the backend that can be used by a process supervisor to ensure that the application is in a healthy state? It can simply return HTTP 204 for okay and 500 otherwise.
This check should ensure that the backend is responding and can reach the database. It should not require authentication.

**Comment by dani** [comment]
Could be usefull if we run several instances of rallly behind a loadbalancer

**Comment by lukevella** [comment]
Any examples of other self-hosted apps implementing this?

**Reply by dani** [reply]
Most web app has something like this. Etherpad Lite is the first that comes to my mind with a /health endpoint. The output on this endpoint should have minimum content (either empty, or a simple OK), as some system like consul register it in a db at each check. The important part is just to respond with a 20x code on success, and a 5xx on failure.

**Comment by icsy7867** [comment]
I will add here, I recently configured our zabbix instance to monitor the JSON health page for a container i have running in kubernetes.
https://developer.1password.com/docs/connect/connect-api-reference/

So this is just one example....

No auth needed for this endpoint:
https://app.my.company.com/health

And this returns:

```
{"name":"1Password Connect API","version":"1.7.2","dependencies":[{"service":"sqlite","status":"ACTIVE","message":"Connected to /home/opuser/.op/data/1password.sqlite"},{"service":"account_data","status":"AVAILABLE","message":"Account data is available"},{"service":"sync","status":"ACTIVE"},{"service":"1Password","status":"ACTIVE"}]}
```

**Reply by lukevella** [reply]
Thank you, that's very useful!
